slam-ws/src/nav_rtab/launch/rtab.launch

```diff
@@ -0,0 +1,28 @@
+<launch>
```

Comment:
Should call `move_base.launch` with the correct configurations (ie. the same configurations gmapping uses for now) or create `rtab_navigation.launch` which calls `rtab.launch` and `move_base.launch`